docs/Feature Documentation/Audio and Video Streaming Setup/index.md

```diff
@@ -170,33 +115,33 @@ gst-launch-1.0 souphttpsrc location=http://127.0.0.1:5050 ! "application/x-rtp-s
 gst-launch-1.0 souphttpsrc location=http://127.0.0.1:5080 ! audio/x-raw,format=S16LE,rate=16000,channels=1 ! pulsesink
 ```
 
-# Start SDL Core
-In the build folder
+## Video Streaming States
 
-```
-cd bin
-./start.sh
-```
-!!! NOTE
-If you want to use a USB connection, you must run
-```
-sudo ./start.sh
-```
-!!!
+This section describes how Core manages the streaming states of mobile applications. Only one application may stream video at a time, but audio applications may stream while in the LIMITED state with other applications.
+
+When an app is moved to HMI level `FULL`:
+- All non-streaming applications go to HMI level `BACKGROUND`
+- All apps with the same App HMI Type go to `BACKGROUND`
+- Streaming apps with a different App HMI Type that were in `FULL` go to `LIMITED`
+
+When an app is moved to HMI level `LIMITED`:
+- All non-streaming applications keep their HMI level
+- All applications with a different App HMI Type keep their HMI level
+- Applications with the same App HMI Type go to `BACKGROUND`
 
-### Start the web HMI
+### Start the Web HMI
 
 CD into the HMI repository and run
 ```
 chromium-browser index.html
 ```
 
-### Start Video or Audio Stream
+### Additional Resources
 
 !!! NOTE
-No open source mobile application currently implements video streaming.
+Livio provides an [example video streaming android application](https://github.com/livio/sdl_video_streaming_android_sample).
```

Comment:
```suggestion
Livio provides a [sample video streaming android application](https://github.com/livio/sdl_video_streaming_android_sample). This app is for testing purposes only.
```

Reply:
sample is in the name of the app so i like example more, and testing purposes only is in the readme, why do you think we should add here too? i think that might unnecessarily scare people off from using it as a reference.

Reply:
I was thinking it would make it clearer but yeah I guess `example` should make it apparent that the app is only meant for testing. The change might be uneccesary